packages/cyberstorm/src/components/componentStyles/Button.module.css

```diff
@@ -0,0 +1,97 @@
+.button {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  align-items: center;
+  flex: none;
+  gap: var(--gap-medium);
+
+  padding: var(--padding-medium-small) var(--padding-medium);
+  border-radius: var(--size-small);
+
+  background-clip: padding-box, border-box;
+  --bg-color: var(--color-surface--3);
+  background-color: var(--bg-color);
+  --text-color: var(--color-white);
+  color: var(--text-color);
+}
+
+.button__default {
+  --text-color: var(--color-white);
+  --bg-color: var(--color-surface--3);
+}
+.button__default:hover {
+  --bg-color: var(--color-surface--4);
+  outline: 2px solid var(--color-border-2);
+}
+
+.button__defaultDark {
+  --text-color: var(--color-white);
+  --bg-color: var(--color-surface--1);
+}
+
+.button__defaultDark:hover {
+  --bg-color: var(--color-surface--2);
+  outline: 2px solid var(--color-border-2);
+}
+
+.button__primary {
+  --text-color: var(--color-black);
+  --bg-color: var(--color-cyber-green);
+}
+
+.button__primary:hover {
+  --bg-color: var(--color-cyber-green--40);
+}
+
+.button__danger {
+  --text-color: var(--color-white);
+  --bg-color: var(--color-red);
+}
+
+.button__danger:hover {
+  --bg-color: var(--color-red--40);
+}
+
+.button__defaultWithBorder {
+  --bg-color: var(--color-surface--3);
+  --text-color: var(--color-white);
+  outline: 2px solid var(--color-border-2);
+}
+
+.button__defaultWithBorder:hover {
+  --bg-color: var(--color-surface--4);
+  outline: 2px solid var(--color-border-3);
+}
+
+.button__specialGreen {
```

Comment:
FYI the storybook shows odd corners for these buttons in Firefox: 
![image](https://user-images.githubusercontent.com/3500016/197724950-8485d711-1045-4a8f-883c-af685dca219f.png)

Reply:
This is due to this rule `border-image-source: var(--color-gradient-purple-green);` present in the styles, which basically creates a border on top of the bg-image hack which shows like this on Firefox. Setting the border-image is unnecessary and should be removed. See the [Codepen](https://codepen.io/gravija/pen/poKzGYJ) for a working example.

Reply:
Using the example now